run_tests.py

```diff
@@ -2,4 +2,5 @@
 
 import diffpy.pdfgui.tests
```

Comment:
per cookiecutter, we should delete this file.

Reply:
Yes we need to delete this if we move on to pytest